plugins/com.google.gcp.eclipse.appengine.localserver.test/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,10 @@
+Manifest-Version: 1.0
+Bundle-ClassPath: .
+Bundle-ManifestVersion: 2
+Bundle-Name: Test
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Bundle-SymbolicName: com.google.gcp.appengine.localserver.test
```

Comment:
This name doesn't match the directory and pom.xml artifact name: it's com.gcp.**eclipse**.appengine.localserver.test